docs/faq.md

```diff
@@ -399,3 +399,25 @@ You are highly encouraged to avoid changes such as the ones above to questions w
 Neither of these options will affect the score a student may already have obtained in any previous variants. However, any work a student may have started on an open variant but not yet submitted will be lost.
 
 For exams and other summative assessments where students may have been negatively impacted by such a change, you are encouraged to consider [giving students credit for issues such as these](regrading.md).
+
+## When I open some of the CSV downloads, some of the data is in the wrong columns
+
+When loading some assessment download files, such as `*_all_submissions.csv`, in Excel, you may notice that some data appears to be in the wrong column or that some lines wrap incorrectly. This can happen for two reasons:
+
+1. **Cell Size Limit:** Excel has a maximum cell size of 32,767 characters. If a column, such as `Params`, contains values exceeding this limit, Excel will truncate or fail to display the data correctly.
+2. **CSV Parsing and Escaping Issues:** The `Params` column often contains JSON data, which includes special characters like quotes, commas, and newlines. Even when properly escaped in the CSV file, Excel’s parser may not handle these cases correctly, leading to data misalignment or unexpected formatting.
+
+To work with these files, you can use Pandas in Python. This allows you to either save a new CSV without the `Params` column for easier processing in Excel, export the data in JSON format, or process it directly in Python. For example:
```

Comment:
The issue may be in other columns (true answer, submitted answer, feedback), although I agree params is the most likely culprit. Maybe clarify that?

Reply:
https://github.com/PrairieLearn/PrairieLearn/pull/11280/commits/42e2fee4fc18868b7fb1cb1241fa6b6aa88da440